### ISSUE TITLE
missing dependency `matplotlib`

### ISSUE THREAD

**Issue by Kr1ss-XD** [GuidoBartoli/sherloq#51]
Hi,

I suppose that `requirements.txt` might be missing a `matplotlib` entry : 

After installing `sherloq` in a virtual Python environment according to the [instructions](https://github.com/GuidoBartoli/sherloq/blob/master/README.md#linux), starting sherloq fails, yielding the following exception : 

```sh
( sq)[ master|/dev/shm/sherloq/gui/]
$ python sherloq.py
Traceback (most recent call last):
  File "/dev/shm/sherloq/gui/sherloq.py", line 20, in <module>
    from histogram import HistWidget
  File "/dev/shm/.venv/sq/lib/python3.9/site-packages/shiboken2/files.dir/shibokensupport/__feature__.py", line 142, in _import
    return original_import(name, *args, **kwargs)
  File "/dev/shm/sherloq/gui/histogram.py", line 17, in <module>
    from matplotlib.backends.backend_qt5agg import FigureCanvas
  File "/dev/shm/.venv/sq/lib/python3.9/site-packages/shiboken2/files.dir/shibokensupport/__feature__.py", line 142, in _import
    return original_import(name, *args, **kwargs)
ModuleNotFoundError: No module named 'matplotlib'
```

After

```sh
( sq)[ master|/dev/shm/sherloq/gui/]
$ pip install matplotlib
```

inside the venv, I can successfully start Sherloq.

---

**Environment :**

OS | python | virtualenv | virtualenvwrapper | shiboken2 | sherloq
:---:|:---:|:---:|:---:|:---:|:---:
Archlinux | 3.9.9 | 20.8.0 | 4.8.4 | 5.15.2 | [master]

**Comment by GuidoBartoli** [comment]
I used `pip-chill` to reduce the number of required packages listing only the non dependant one inside `requirements.txt` file, but somehow `matplotlib` was not included. I just fixed it in abaebe8 commit, it should work now, thanks for pointing that out!

**Comment by Kr1ss-XD** [comment]
Thanks very much for the quick response @GuidoBartoli !
